features/fixtures/sampler/index.js

```diff
@@ -8,6 +8,8 @@ config.delivery = new StandardDelivery(endpoint, endpoint)
 config.autoCaptureSessions = false
 const _bugsnag = new Client(config)
 
+export default _bugsnag;
```

Comment:
Why are all these named with underscores? In JS a `_` prefix is used to denote "private" APIs by convention (where it can't be enforced).

Unless there is a reason for this, I think this identifier should just have the `_` removed wherever it is used.

Reply:
Good to know - it was named like this before when it was only used in this file. I'll update the identifier